d2common/d2util/debug_print.go

```diff
@@ -37,16 +37,16 @@ type GlyphPrinter struct {
 // Basic Latin and C1 Controls and Latin-1 Supplement.
 //
 // DebugPrint always returns nil as of 1.5.0-alpha.
-func (p *GlyphPrinter) Print(target *ebiten.Image, str string) error {
-	p.PrintAt(target, str, 0, 0)
+func (p *GlyphPrinter) Print(target interface{}, str string) error {
+	p.PrintAt(target.(*ebiten.Image), str, 0, 0)
 	return nil
 }
 
 // PrintAt draws the string str on the image at (x, y) position.
 // The available runes are in U+0000 to U+00FF, which is C0 Controls and
 // Basic Latin and C1 Controls and Latin-1 Supplement.
-func (p *GlyphPrinter) PrintAt(target *ebiten.Image, str string, x, y int) {
-	p.drawDebugText(target, str, x, y, false)
+func (p *GlyphPrinter) PrintAt(target interface{}, str string, x, y int) {
+	p.drawDebugText(target.(*ebiten.Image), str, x, y, false)
```

Comment:
this is from #1014 , you should pull+rebase off of master instead of duplicating that work inside of this PR.